### PR TITLE
Adds a init method to EmeraldPinCodeView.kt

### PR DESCRIPTION
**Pull request by rafael-sc**
#### What this PR does?
Adds a init method to EmeraldPinCodeView.kt

#### How should it be manually tested?
Scrolldown on mainActivity and click on PinView

### CODE REVIEW

File: emeraldcomponents/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ def deployKey = System.env.PACKAGECLOUD_TOKEN != null ? System.env.PACKAGECLOUD_
 def fileFilter = ['**/R.class', '**/R$*.class', '**/BuildConfig.*', '**/Manifest*.*', '**/*Test*.*', '**/*$*', 'android/**/*.*', '**/*Function*', '**/*_impl*']
 def debugTree = fileTree(dir: "$project.buildDir/tmp/kotlin-classes/debug", excludes: fileFilter)
 def mainSrc = "$project.projectDir/src/main/kotlin"
-def version = '1.14.0'
+def version = '1.14.1'
```
Comment (renanrbs) [inline emeraldcomponents/build.gradle:18]: only change versions on release pr

Comment (rafael-sc) [reply emeraldcomponents/build.gradle:18]: oh, sorry.

File: emeraldcomponents/src/main/java/br/com/stone/emeraldcomponents/basic/pinview/EmeraldPinCodeView.kt [hunk 2]
```diff
@@ -46,6 +46,17 @@ class EmeraldPinCodeView @JvmOverloads constructor(
         }
     }
 
+    fun init(isNumeric: Boolean, itemCount: Int) {
```
Comment (renanrbs) [inline emeraldcomponents/src/main/java/br/com/stone/emeraldcomponents/basic/pinview/EmeraldPinCodeView.kt:49]: you could also use this method on line 43

File: emeraldcomponents/src/main/java/br/com/stone/emeraldcomponents/basic/pinview/EmeraldPinCodeView.kt [hunk 3]
```diff
@@ -46,6 +46,17 @@ class EmeraldPinCodeView @JvmOverloads constructor(
...
+        resetPinView()
+        setEditTextList(createItems(itemCount, isNumeric))
```
Comment (renanrbs) [inline emeraldcomponents/src/main/java/br/com/stone/emeraldcomponents/basic/pinview/EmeraldPinCodeView.kt:51]: is this behavior tested?

Comment (rafael-sc) [reply emeraldcomponents/src/main/java/br/com/stone/emeraldcomponents/basic/pinview/EmeraldPinCodeView.kt:38]: Just added a test.

**Pull request merged**